packages/flutter/lib/src/material/data_table.dart

```diff
@@ -1047,6 +1082,52 @@ class DataTable extends StatelessWidget {
   }
 }
 
+/// Height settings for DataRows.
+/// Either a fixed row height or a row height that adjusts to its content can be specified.
+@immutable
+class DataTableRowHeight with Diagnosticable {
```

Comment:
class name should match the parameter name unless there is a good reason, i.e. shared between multiple widget